mesh_generators/GenCylGenerators.cs

```diff
@@ -79,38 +80,71 @@ override public MeshGenerator Generate()
             triangles = new IndexArray3i(nSpanTris + nCapTris);
 
             Frame3f fCur = new Frame3f(Frame);
+            Frame3f fNext = new Frame3f(Frame);
             Vector3d dv = CurveUtils.GetTangent(Vertices, 0); ;
             fCur.Origin = (Vector3f)Vertices[0];
             fCur.AlignAxis(2, (Vector3f)dv);
             Frame3f fStart = new Frame3f(fCur);
 
+            float pathLength = WeighedUV ? (float)CurveUtils.ArcLength(Vertices) : 0;
+            float running_uv_along = 0;
+
             // generate tube
             for (int ri = 0; ri < nRings; ++ri) {
 
                 // propagate frame
                 if (ri != 0) {
                     Vector3d tan = CurveUtils.GetTangent(Vertices, ri);
                     fCur.Origin = (Vector3f)Vertices[ri];
-                    if (ri == 11)
```

Comment:
I removed this because it didn't seem to do anything, the `dv` variable isn't used anywhere.